lib/mongoid/association/referenced/has_many/proxy.rb

```diff
@@ -199,7 +199,7 @@ def exists?
           # @note This will keep matching documents in memory for iteration
           #   later.
           #
-          # @param [ Object | Array<Object> ] *args The ids.
+          # @param [ [ BSON::ObjectId | String ]... ] *args The ids.
```

Comment:
You can add the `...` here but the types were correct as written, please put them back the way they were.

Reply:
Fixed